### PR TITLE
#57 About page rework to use yml

### PR DESCRIPTION
**Pull request by spiraldancer**
Updated about page to use yml/jekyll. There are two sections: leadership and staff. Also included a section to build the social links/icons. This will need updated with the correct hrefs and email address for the bios. It currently builds Sara's social as the page does now.

### CODE REVIEW

**Comment by kcjonesevans** [comment]
Can you update based on the conflicts?

**Comment by spiraldancer** [comment]
working on it now

File: _data/about_bios.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        Sara Cope is currently a developer with GSA Digital Service, and CSS
+        guru with over 10 years of experience. She has a B.S. in Management
+        Information Systems from Wright State University. When Sara isn't
+        immersed in all things web, she can be found playing tabletop games
+        and supporting her local park system.
+   image: /img/people/saracope.jpg
+   sociallinks:
+     - label: Email
+       link: mailto:sarac@girldevelopit.com
+       inconclass: fa fa-envelope
```
Comment (k80bowman) [inline _data/about_bios.yml]: could we change this to `iconclass`? I realize there are a lot of them, but I'd rather the name not be confusing. Thanks!

File: _site/about/index.html [hunk 2]
```diff
@@ -7,16 +7,9 @@
...
-    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet" />
-    <script>
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-        ga('create', 'UA-92693101-1', 'auto');
-        ga('send', 'pageview');
-    </script>
```
Comment (k80bowman) [inline _site/about/index.html:19]: Thanks for moving all of these to the bottom :)

File: _site/css/application.css [hunk 3]
```diff
@@ -654,6 +686,10 @@ footer {
   margin-bottom: 3rem;
   text-align: center; }
 
+.dashedLineBelow {
+  border-bottom: 1px dashed #f95a61;
+  padding-bottom: .2rem; }
+
```
Comment (k80bowman) [inline _site/css/application.css:698]: This is nice!

File: _site/css/application.css [hunk 4]
```diff
@@ -245,6 +245,38 @@ hr {
...
+    div.social i:hover {
+      color: #fff; }
+
+div.social_links {
+  margin: auto;
+  width: 50%;
+  text-align: center; }
+  div.social_links i {
+    color: red; }
+    div.social_links i:hover {
```
Comment (k80bowman) [inline _site/css/application.css]: this hover effect is working a little strangely and I'm not sure it's applied to the correct element. The yellow background color hover gets applied when you hover just outside the icon like this:
image

But the icon color doesn't change to white until you actually hover on the icon.

File: about.html [hunk 5]
```diff
@@ -43,78 +43,67 @@ <h2>Our Story</h2>
...
-        <h3>Catherine Meade, Instructor</h3>
-        <div class="profile-links">
-          <a target="_blank" href="https://twitter.com/catheraaine" aria-label="Catherine Meade Twitter"><i class="profile-icon fa fa-twitter"></i></a>
-          <a target="_blank" href="https://www.linkedin.com/in/catheraaine/" aria-label="Catherine Meade LinkedIn"><i class="profile-icon fa fa-linkedin-square"></i></a>
-          <a target="_blank" href="http://catheraaine.com" aria-label="Catherine Meade Website"><i class="profile-icon fa fa-globe"></i>
-        </div>
-        <p>
-          Catherine is a web developer at Sparkbox, and teaches Intro to HTML/CSS with GDI. She has a B.S. in Art Education and a strong passion for teaching. Cat's other favorite things include reading, video games, tabletop games, oil painting, and boffer sports.
-        </p>
+      {% for currBio in site.data.about_bios.staff %}
```
Comment (k80bowman) [inline about.html:79]: Somehow Kevin's bio is being labeled "profile-left", which is why I think we're seeing this:
image

**Review by k80bowman — CHANGES_REQUESTED** [review]
It's looking really nice, thanks for making all of these changes. I do have a couple of requests, though, before I can merge this in. Thanks again!

**Review by k80bowman — APPROVED** [review]

**Comment by k80bowman** [comment]
These changes were merged using a different branch.

**Pull request closed without merging**